web/template/partial/stream/chat.gohtml

```diff
@@ -1,14 +1,36 @@
 {{define "chat"}}
     {{- /*gotype: TUM-Live/web.WatchPageData*/ -}}
     {{$stream := .IndexData.TUMLiveContext.Stream}}
-    <div id="chatWrapper"
-         class="hidden md:flex flex-col w-full md:h-full border-t-2 mt-2 md:mt-0 md:border-t-0 md:dark:border-l dark:border-secondary dark:bg-secondary-lighter bg-gray-100 md:w-2/6 lg:w-1/6 text-1 pt-3">
-        <div id="chatBox" class="overflow-scroll overflow-x-hidden overflow-y-scroll" style="flex: 2;">
+    <script src="/static/assets/ts-dist/chat.js?v={{.IndexData.VersionTag}}"></script>
+    <div x-data="{show: false}" x-init="show = initChat();" id="chatWrapper"
```

Comment:
This should help with the flickering when loading the website (alpine hides the chat until it's done initializing the page: https://alpinejs.dev/directives/cloak)

```suggestion
    <div x-cloak x-data="{show: false}" x-init="show = initChat();" id="chatWrapper"
```

Reply:
Thanks! Works like a charm.